src/wrapper.rs

```diff
@@ -8,10 +8,25 @@ use syn::{
     Type,
 };
 
-#[derive(Eq, PartialEq, Clone, Copy)]
-pub enum GenOpt {
-    All,
-    NoImplMessage,
+bitflags! {
```

Comment:
I doubt that whether we really need so many options or not. I only like `MSG`, `CLR`, `HAS`, and `TKE` in person. `NEW` can be replaced with `Default`, and `GET`, `SET` and `MUT` are useless because we can touch the internal fields directly.

Reply:
The process of eliminating wrapper usages should be step by step and this (relatively) fine grained config can help us do that.

Reply:
Or can we merge these 3 as one option?

Reply:
@hicqu I guess you've missed the point of this PR. What I was trying to do is to introduce a mechanism which can gradually reduce the wrapper functions. The ultimate goal is passing a `GenOpt::empty()` to the wrapper generator. Currently we're generating all of them so we can quickly launch prost support and my plan is to replace all usages of the generated wrappers with direct access. These options allow me to eliminate the generated wrappers step by step. That's all. :wink:

Reply:
So after all wrappers are removed, should we remove these options also?

Reply:
We can! And I believe we're supposed to do so.